main-service/src/main/java/ru/practicum/events/event_models/Event.java

```diff
@@ -0,0 +1,76 @@
+package ru.practicum.events.event_models;/* # parse("File Header.java")*/
+
+import lombok.*;
+import ru.practicum.category.category_models.Category;
+import ru.practicum.users.user_models.User;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+/**
+ * File Name: Event.java
+ * Author: Marina Volkova
+ * Date: 2023-10-20,   6:59 PM (UTC+3)
+ * Description:
+ */
+
+@Entity
+@Table(name = "events")
+@Data
```

Comment:
В аннотации Data есть реализация методов equals & hashCode и это очень важные методы для сущности БД, поэтому не стоит отдавать их реализацию на откуп сторонней библиотеки ломбок, а лучше сделать их самостоятельно. В целом  достаточно вместо data ставить Getter & Setter

Подробнее здесь:
https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/

ключевое ниже

Your equals() method should return false if the primary key value of both entity objects is null. But Lombok’s equals() method returns true. Because of that, you can’t add two new entity objects to a Set

Reply:
Удалена аннотация @Data.